mix.exs

```diff
@@ -32,7 +32,7 @@ defmodule Kitto.Mixfile do
     [{:cowboy, "~> 1.0.0"},
      {:plug, "~> 1.3"},
      {:poison, "~> 3.0"},
-     {:fs, "~> 2.11.0"},
+     {:fs, github: "synrc/fs"},
```

Comment:
Honestly with the repo not having git tags to match the released versions its quite hard to figure out which are the unreleased updates you're referring to. I prefer to downgrade to 0.9.2 than re-add a github dependency.

I've commented on the lack of git tags, see https://github.com/synrc/fs/issues/24#issuecomment-266221803.